be/src/runtime/data_stream_mgr.cpp

```diff
@@ -145,6 +145,8 @@ Status DataStreamMgr::transmit_chunk(const PTransmitChunkParams& request, ::goog
         // in acquiring _lock.
         // TODO: Rethink the lifecycle of DataStreamRecvr to distinguish
         // errors from receiver-initiated teardowns.
+        LOG(WARNING) << request.sender_id() << " sender transmits chunks to a non-existing receiver fragment "
```

Comment:
I think this is a normal case when FE sends cancel, why do you print the WARNING log here?

Reply:
change it to vlog_query.